client/my-sites/earn/main.tsx

```diff
@@ -38,9 +38,12 @@ const EarningsMain = ( { section, query, path }: EarningsMainProps ) => {
 	const adsProgramName = isJetpack ? 'Ads' : 'WordAds';
 
 	const layoutTitles = {
-		earnings: translate( '%(wordads)s Earnings', { args: { wordads: adsProgramName } } ),
-		settings: translate( '%(wordads)s Settings', { args: { wordads: adsProgramName } } ),
-		payments: translate( 'Recurring Payments' ),
+		'ads-earnings': translate( '%(wordads)s Earnings', { args: { wordads: adsProgramName } } ),
+		'ads-settings': translate( '%(wordads)s Settings', { args: { wordads: adsProgramName } } ),
+		'ads-payments': translate( '%(wordads)s Payments', { args: { wordads: adsProgramName } } ),
+		payments: translate( 'Payment Settings' ),
+		customers: translate( 'Customers' ),
+		stats: translate( 'Stats' ),
```

Comment:
A nit but we should probably move to Sentence case rather than Title Case for these Earn section headings? @crisbusquets for advice.